src/AssetManager.Web/Startup.cs

```diff
@@ -21,6 +23,21 @@ public Startup(IConfiguration configuration)
 
         public IConfiguration Configuration { get; }
 
+        public void ConfigureServices(IServiceCollection services)
```

Comment:
There is big problem with your code here. There was already `ConfigureServices` method that get called by `Configure[environment]Services` , example `ConfigureDevelopmentServices` . Don't blindly copy paste any codes. 

> Continue next comment